<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QWidget" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>480</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="button_vpred">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>510</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Vpřed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_zpet">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>510</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Zpět</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_promote">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>510</y>
     <width>76</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Dáma</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Věž</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Střelec</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Jezdec</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="button_auto">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>510</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBox_auto">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>510</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_nacti">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>490</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Načíst</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_uloz">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>520</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Uložit</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>161</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Povýšit na</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_sachy">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Šachy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_dama">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Dáma</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
